frontend/packages/otr/src/redux/reducers/clerkInterpreter.ts

```diff
@@ -1,14 +1,32 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { APIResponseStatus } from 'shared/enums';
 
+import { QualificationStatus } from 'enums/interpreter';
 import {
   ClerkInterpreter,
+  ClerkInterpreterFilters,
   ClerkInterpreterState,
 } from 'interfaces/clerkInterpreter';
 
 const initialState: ClerkInterpreterState = {
   interpreters: [],
   status: APIResponseStatus.NotStarted,
+  filters: {
+    qualificationStatus: QualificationStatus.Effective,
+  },
+  qualificationLanguages: [],
+};
+
+const getQualificationLanguages = (interpreters: Array<ClerkInterpreter>) => {
+  const languages = new Set<string>();
+  interpreters.forEach(({ qualifications }) => {
+    qualifications.forEach(({ fromLang, toLang }) => {
+      languages.add(fromLang);
+      languages.add(toLang);
+    });
+  });
+
+  return Array.from(languages);
 };
```

Comment:
Tarviiko tätä ollenkaan? Tästä voi tulla raskas operaatio, kun datan määrä kasvaa. Miksi ei listata kaikki mahdolliset kielet, varsinkin kun lähtökieli on vain suomi tällä hetkellä.

Reply:
Varsinkin kun tiedetään että lähtökieli on tällä hetkellä vain suomi

Reply:
Ajattelin että olisi käyttäjäystävällisempi näin ja analoginen sen kanssa miten AKT:ssa kielivalinnat näytetään.

Reply:
Se miten raskas operaatio tämä on käy toki parhaiten ilmi oikealla datalla. Lähtökohtaisesti en pitäisi ihan mahdottomana, koska eihän tulkkeja / pätevyyksiä kovin isoa määrää ole - tällä hetkellä taidetaan liikkua muutamassa sadassa tulkissa.

Reply:
Joo, AKT:ssä backend palauttaa niitä kielipareja, mutta tässä taas niitä ei palauteta. Mistä kenttä tulee olemaan hyvin todennäköisesti vain suomi + ehkä ruotsi. En kyllä laskisi enkä laittaisi niitä kielejä redux tilaan ellei ole pakko. Toki tuohon vaikuttaa se, haluaako virkailijat, että näytetään vain järjestelmästä löytyviä kielejä..